crates/re_log/src/channel_logger.rs

```diff
@@ -0,0 +1,52 @@
+//! Capture log messages and send them to some receiver over a channel.
+
+pub struct LogMsg {
+    pub level: log::Level,
+    pub msg: String,
+}
+
+/// Pipe log messages to a channel.
+pub struct ChannelLogger {
+    filter: log::LevelFilter,
+    tx: parking_lot::Mutex<std::sync::mpsc::Sender<LogMsg>>,
+}
+
+impl ChannelLogger {
+    pub fn new(filter: log::LevelFilter) -> (Self, std::sync::mpsc::Receiver<LogMsg>) {
+        let (tx, rx) = std::sync::mpsc::channel();
+        (
+            Self {
+                filter,
+                tx: tx.into(),
+            },
+            rx,
+        )
+    }
+}
+
+impl log::Log for ChannelLogger {
+    fn enabled(&self, metadata: &log::Metadata<'_>) -> bool {
+        if metadata.target().starts_with("wgpu") || metadata.target().starts_with("naga") {
```

Comment:
feels like we have this hardcoded in too many places now :/
Can we have the `LOUD_CRATES` logic/"database" move to a central place?

Reply:
Or I spend a couple of hours fixing wgpu/naga…